src/components/AbstractPageDefinitions.js

```diff
@@ -436,6 +442,70 @@ export default class AbstractPageDefinitions extends Component {
     )
   }
 
+  showVersionSelectorPopup(component, multiple) {
+    this.setState({ showVersionSelectorPopup: true, multipleVersionSelection: multiple, selectedComponent: component })
+  }
+
+  applySelectedVersions(versions) {
+    const { multipleVersionSelection, selectedComponent } = this.state
+    if (!multipleVersionSelection) {
+      return this.setState({ showVersionSelectorPopup: false }, async () => {
+        if (selectedComponent.changes) {
+          const key = `open${Date.now()}`
+          const NotificationButtons = (
```

Comment:
this should be factored out as a separate function that can be reused. Also simplifies this code making it easier to understand

Reply:
Actually there was already a reusable component. I missed to use it in this point, thanks for catching it